app/views/shared/forms/_text_fields.html.erb

```diff
@@ -6,8 +6,10 @@
 <div class="govuk-form-group <%= group_error_class %>">
   <%= form.label field_name, class: ['govuk-label'] %>
   <% hint = t(".#{field_name}_hint", default: '') %>
-    <% if hint.present? %>
-      <%= content_tag(:span, hint, id: "#{field_name}-hint".dasherize, class: ['govuk-hint']) %>
+    <% if local_assigns[:hint?] == true %>
```

Comment:
💭 This block can probably be simplified to something like the following:

```ruby
<% if local_assigns[:hint?] && hint.present? %>
  <%= content_tag(:span, hint, id: "#{field_name}-hint".dasherize, class: ['govuk-hint']) %>
<% end %>
```

Reply:
❓ Is the expected behaviour that by default the hint is not shown for **all text fields** unless we specifically set `hint?: true` in the form for those fields? Even when the hint text for those text fields are defined in the locale file?

Reply:
I think it might be better if we flipped this logic, so that hints are shown by default and only hidden if explicitly set by the developer - I think this situation is a minority case, in most cases if a hint is being defined then it probably should be shown.

Reply:
Reversed the logic here.
Local is now called hide_hint and will hide the text field hint when set to true.